docs/pages/documentation/3-querying-with-graql/define-queries.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Define Queries
+keywords: graql, query, define
+last_updated: August 10, 2016
+tags: [graql]
+summary: "Graql Define Queries"
+sidebar: documentation_sidebar
+permalink: /documentation/graql/define-queries.html
+folder: documentation
+---
+
+The page documents use of the Graql `define` query, which will define a specified [variable pattern](#variable-patterns)
```

Comment:
`The page documents use of the Graql `define` query, which will define a specified [variable pattern](#variable-patterns) describing the schema`

Short sentences are better and also if we ever rebrand again I won't need to hunt dow this one use case of "knowledge base"